infra/app/app-config/env-config/variables.tf

```diff
@@ -10,3 +10,7 @@ variable "environment" {
 variable "has_database" {
   type = bool
 }
+
+variable "has_incident_management_service_integration" {
```

Comment:
nit: we could probably simplify to `has_incident_management_service` to be consistent with `has_database`:
```suggestion
variable "has_incident_management_service" {
```

Reply:
done